docs/defaults-detailed.rst

```diff
@@ -183,7 +195,7 @@ General interface parameters
   - ``auto``: bring the interface up at boot time by the ``networking``
```

Comment:
Yes, there's default value, `boot` - at least for most systems. I'll add short explanation here how the role defines it if it's not specified.